Vostok.Applications.Scheduled/ScheduledActionsBuilder.cs

```diff
@@ -60,5 +64,18 @@ internal ScheduledActionsRunner BuildRunnerInternal()
                 return Task.CompletedTask;
             };
         }
+
+        private static TArg ExtractArgumentFromOnDemandScheduler<TArg>(IScheduler scheduler)
+        {
+            if (scheduler is OnDemandSchedulerWithArgument<TArg> onDemandScheduler)
+                return onDemandScheduler.GetLastArgumentValue();
+
+            throw new NotSupportedException("Argument passing is supported for OnDemandScheduler only.");
```

Comment:
Would an extension that explicitly accepts `OnDemandSchedulerWithArgument` instead of `IScheduler<Arg>` solve this?

Reply:
Actually, no, because `OnDemandSchedulerWithArgument<TArg>` is internal, so we can't expose it in public methods.

However, I gave it a thought one more time, and I think that it's better to remove `IScheduler<TArg>`.